File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/javascript/pyramsd.js
function FizzBuzz(num){
    for (let i = 1; i <= 100; i++){
        if (i % 3 == 0){
            console.log("fizz");
        }if (i % 5 == 0){
            console.log("buzz");
        }if (i % 3 == 0 && i % 5 == 0){
            console.log("fizzbuzz");
        }else{
            console.log(i);
        }
    }
}

FizzBuzz(100);

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/c++/pyramsd.cpp
#include <iostream>
using namespace std;

int main(){
    for (int i = 0; i <= 100; i++){
        if (i % 3 == 0 && i % 5 == 0) cout << "fizzbuzz" << endl;
        else if (i % 3 == 0) cout << "fizz" << endl;
        else if (i % 5 == 0) cout << "buzz" << endl;
        else cout << i << endl;
    }
}
